addons/packages/vsphere-csi/2.6.2/bundle/config/values.yaml

```diff
@@ -12,7 +12,7 @@ daemonset:
   updateStrategy: null
 vsphereCSI:
   tlsThumbprint: ""
-  namespace: kube-system
+  namespace: vmware-system-csi
```

Comment:
where are we creating `vmware-system-csi` namespace?

Reply:
handling will be done in tanzu-framework